content/doc/book/pipeline/getting-started.adoc

```diff
@@ -72,6 +72,10 @@ Jenkins supports entering Pipeline directly into the classic UI, it is
 generally considered best practice to define the Pipeline in a `Jenkinsfile`
 which Jenkins will then load directly from source control.
 
+This video provides basic instructions on how to write both Declarative and Scripted pipelines.
```

Comment:
```suggestion
This video provides basic instructions on how to write both Declarative and Scripted Pipelines.
```

Reply:
I believe Pipeline is capitalized in this context.

Reply:
Makes sense, I've gone ahead and updated things accordingly!